MiniSpace.Services.Identity/src/MiniSpace.Services.Identity.Api/Program.cs

```diff
@@ -69,16 +69,6 @@ public static async Task Main(string[] args)
                             await ctx.RequestServices.GetService<IRefreshTokenService>().RevokeAsync(cmd.RefreshToken);
                             ctx.Response.StatusCode = 204;
                         })
-                        .Post<GrantOrganizerRights>("users/{userId}/organizer-rights", async (cmd, ctx) =>
-                        {
-                            await ctx.RequestServices.GetService<IIdentityService>().GrantOrganizerRightsAsync(cmd);
-                            ctx.Response.StatusCode = 204;
-                        })
-                        .Delete<RevokeOrganizerRights>("users/{userId}/organizer-rights", async (cmd, ctx) =>
-                        {
-                            await ctx.RequestServices.GetService<IIdentityService>().RevokeOrganizerRightsAsync(cmd);
-                            ctx.Response.StatusCode = 204;
-                        })
                         .Post<BanUser>("users/{userId}/ban", async (cmd, ctx) =>
                         {
                             await ctx.RequestServices.GetService<IIdentityService>().BanUserAsync(cmd);

```

Comment:
Why revoke-organizer-rights were removed?

Reply:
We decided to resign from organizing implementing in that way.